examples/Swift/Sample/TailLoadingCellNode.swift

```diff
@@ -55,7 +55,10 @@ final class SpinnerNode: ASDisplayNode {
   }
 
   override init() {
-    super.init(viewBlock: { UIActivityIndicatorView(activityIndicatorStyle: .Gray) }, didLoadBlock: nil)
+    super.init()
+    setViewBlock {
+        UIActivityIndicatorView(activityIndicatorStyle: .Gray)
+    }
```

Comment:
This is a minor breaking API change for Swift. I will raise it on Slack for comment but we shouldn't fret too much.